pkg/v1/tkg/clusterclient/clusterclient.go

```diff
@@ -486,6 +486,8 @@ func (c *client) WaitForClusterInitialized(clusterName, namespace string) error
 	var lastReason string
 	unchangedCounter := 0
 	interval := 15 * time.Second
+	maxTimeout := 3 * c.operationTimeout
```

Comment:
We should add a comment to explain why we added this timeout.

Reply:
sure, updated